.github/workflows/reusable-docker-arm-build.yaml

```diff
@@ -144,8 +145,11 @@ jobs:
           TAG: "latest"
           COSIGN_YES: true
         run: |
-          docker push "$IMAGE:$TAG"
-          image_ref=$(docker image inspect --format='{{index .RepoDigests 0}}' "$IMAGE:$TAG")
+          # Avoid pushing a new image for every commit (re-use latest)
+          export _IMG=$(cat build/IMAGE)
+          export _LATEST=$(echo $_IMG | cut -f1 -d:):latest
```

Comment:
didnt we say that we were not using latest? We should push to master or a similar tag no?

Reply:
We decided to not fix this in this PR (which is already too big). We tried to change as little as possible.